spec/workers/send_chase_email_to_referees_worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe SendChaseEmailToRefereesWorker do
```

Comment:
Can we create a system spec instead to test this?

Reply:
Sure, but roughly how would that look? The system specs seem to largely be user journey focussed. I'm not 100% sure how to test a job that way.

Reply:
Roughly:

- Given a candidate submits an application (create an application with references)
- When there's no response from the referees within 5 days (use timecop to call `SendChaseEmailToRefereesWorker` 5 days later) 
- The referee gets an email with a reminder (check email contents)

I think something similar to the [reject by default test](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/provider_interface/provider_receives_email_when_a_new_application_has_been_submitted_spec.rb#L7-L13) would be nice. Perhaps you can [use some stuff from the referee test as well](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/referee_interface/referee_refuses_to_give_a_reference_spec.rb#L9-L12).

Reply:
Have done 👍